### DISCUSSION TITLE
Plot sizes in the future

### DISCUSSION THREAD

**Discussion by arklas5025** [Chia-Network/chia-blockchain#525] [General]
Hello,

I was reading about the minimum plot size have been changed to k=32
Is it meaning, we have no longer chance to use smaller plots in the future at all? 
I mean if I would use my unused HDD space or USB flash drives, SD cards or old hard drives, I couldn't do it anymore.  I am not sure, is this change is going to be forever or only on the Test Network.  I wonder if could be a pool, for the farmers with smaller plot sizes, and could be a pool for a Bigger Farmers. 

Thank you soo much, 

Laszlo Arkosi

**Comment by hoffmang9** [comment]
No plot smaller than k=32 will be eligible to win on mainnet.

**Reply by Tavanaie** [reply]
Ah...
Its really bad news
there should be more flexible plotting plans!
my SSD Hard Drive size is 238 GB, but minimum possible  temporary space is 332 GB
there is too much difference between 1.8 and 332 GigaBytes

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
